pkg/subsystem/lists/linux_test.go

```diff
@@ -76,8 +76,27 @@ symlinkat(&(0x7f00000004c0)='./file0aaaaaaaaaa/file0\x00', 0xffffffffffffff9c, &
 `),
 				},
 			},
+			// There should be no mm in the subsystems.
 			expect: []string{"ntfs3"},
 		},
+		{
+			name: "dri bug with usb call",
+			crashes: []*subsystem.Crash{
+				{
+					GuiltyPath: `drivers/gpu/drm/udl/udl_drv.c`,
+				},
+				{
+					GuiltyPath: `drivers/gpu/drm/udl/udl_drv.c`,
+					SyzRepro: []byte(`
+# https://syzkaller.appspot.com/bug?id=56e9aec9bc3b5378c9b231a3f4b3329cf9f80990
+# See https://goo.gl/kgGztJ for information about syzkaller reproducers.
+#{"procs":1,"slowdown":1,"sandbox":"none","sandbox_arg":0,"tun":true,"netdev":true,"close_fds":true}
+syz_usb_connect(0x0, 0x24, &(0x7f0000000140)=ANY=[@ANYBLOB="12010000abbe6740e9174e8b089c000000010902120001000000000904000800ff"], 0x0)
+`),
+				},
+			},
+			expect: []string{"dri"},
```

Comment:
There is syz_usb_connect in the repro, why don't we have usb here?

Reply:
It depends on whether we treat e.g. `usb` and `dri` as different or complementary subsystems. If they are treated as different ones, then we have 2 data points (guilty paths for 2 crashes) pointing to `dri` and only one pointing to `usb`.

If it'd be nice to assign `usb` whenever we have the `syz_usb_connect` call in the reproducers, I can mark the subsystem as e.g. `Complementary: true` so that it's always included if mentioned and not considered in subsystem counting. Or just always append subsystems from reproducers.

Reply:
> Or just always append subsystems from reproducers.

Isn't it what Al originally asked for and what started this looking at reproducers?

Reply:
For filesystems, it's different -- `fs` and specific filesystems (e.g. `ext4` or `reiserfs`) are related (parent-child) and, if any specific filesystem is mentioned in a reproducer, `fs` will just get dropped from further consideration.

`dri` and `usb` are parallel to each other.